models/dojo.py

```diff
@@ -0,0 +1,309 @@
+#!/usr/local/bin/python3
+import os
+import random
+
+
+from database_model.database_states import DatabaseManager, OfficeRooms,\
+                                            LivingRooms, Persons
+from models.room.living_space import LivingSpace
+from models.room.office_space import Office
+from models.person.fellow import Fellow
+from models.person.staff import Staff
+
+
+class Dojo(object):
+    """The Dojo class."""
+
+    office_rooms = {}
+    living_rooms = {}
+    persons = {}
```

Comment:
I suggest you refactor to use lists of instances from which you can get a particular instance given the value of one of it's attributes..